package.json

```diff
@@ -15,10 +15,16 @@
     "index.js"
```

Comment:
missing predicate.js in the `files` list, which will cause it to not be published to npm